src/main/cpp/commands/SetControlDrive.cpp

```diff
@@ -0,0 +1,31 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/SetControlDrive.h"
+#include "commands/TankDrive.h"
+#include "commands/SetCamModeDriver.h"
+
+SetControlDrive::SetControlDrive() {
+  AddParallel(new SetCamModeDriver());
```

Comment:
These commands are in the same subsystem. They wont be parallel.

Change to sequential.